ui/src/app/applications/components/utils.tsx

```diff
@@ -493,10 +493,22 @@ function getActionItems(
 
     const resourceActions = getResourceActionsMenuItems(resource, application.metadata, appContext);
 
+    const links = services.applications.getResourceLinks(application.metadata.name, application.metadata.namespace, resource).then(data => {
+        return (data.items || []).map(
+            link =>
+                ({
+                    title: link.title,
+                    iconClassName: 'fa fa-external-link',
```

Comment:
Deep link has optional `iconClass`. It `iconClass` is set we should use it instead of `fa-external-link`